src/TSMapEditor/UI/Windows/TriggersWindow.cs

```diff
@@ -103,6 +103,7 @@ public TriggersWindow(WindowManager windowManager, Map map, EditorState editorSt
         private SelectSpeechWindow selectSpeechWindow;
         private SelectSoundWindow selectSoundWindow;
         private SelectParticleSystemTypeWindow selectParticleSystemTypeWindow;
+        private CreateRandomTriggerSetWindow CreateRandomTriggerSetWindow;
```

Comment:
Private fields are typically lowercase in C#. Rename references accordingly.

```suggestion
        private CreateRandomTriggerSetWindow createRandomTriggerSetWindow;
```